.pre-commit-config.yaml

```
@@ -451,6 +451,12 @@ repos:
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: \.py$
+        exclude: ^dev|^provider_packages|^chart
+      - id: mypy-helm
+        name: Run mypy for helm chart tests
```

```suggestion
      - id: mypy
        name: Run mypy for helm chart tests
```
pre-commit is smart enough to allow duplicate IDs with different file filter configurations. See:  https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/.pre-commit-config.yaml#L29-L139
On the other hand, it makes it easier to run a single check, because you don't need to know all the IDs, but you just need to know 1 ID and the directory you want to check.
```
git ls-files | grep "^chart" | xargs pre-commit run mypy --files
```